packages/ms-authenticator/src/types/IMsAuthenticator.ts

```diff
@@ -0,0 +1,57 @@
+import { IAgentContext, IPluginMethodMap } from '@veramo/core'
+import { AccountInfo } from '@azure/msal-common'
+
+export interface IMsAuthenticator extends IPluginMethodMap {
+  authenticateMsVcApi(): Promise<IMsAuthenticationResponse>
+}
+
+export interface IMsAuthenticationWrapperArgs {
+  authenticationType: MsAuthenticationTypeEnum
+  authenticationArgs:
+    | IMsAuthenticationClientCredentialArgs
+    | IMsAuthenticationUsernamePasswordArgs
+    | IMsAuthenticationAuthorizationCodeArgs
+    | IMsAuthenticationOnBehalfOfArgs
+}
+
+export interface IMsAuthenticationArgs {
+  azClientId: string
+  azTenantId: string
+}
+export interface IMsAuthenticationClientCredentialArgs extends IMsAuthenticationArgs {
+  azClientSecret: string
+  credentialManifest: string
```

Comment:
This probably is a URL right?

Reply:
correct. the code doc specified the format of that url